pkg/cmd/env/templates/azure.tmpl

```diff
@@ -22,11 +24,13 @@ set -e AZURE_CLIENT_ID;
 set -e AZURE_CLIENT_SECRET;
 set -e AZURE_TENANT_ID;
 set -e AZURE_SUBSCRIPTION_ID;
+set -e AZURE_CONFIG_DIR;
```

Comment:
Maybe I'm too early with my review (PR is still marked as draft) and you planned to do the following anyhow:

> Since these actions are persistent, they should be undone when closing the shell or changing the target using the --unset flag.

Shouldn't the config dir be removed by the unset script?

Reply:
✔️